Dockerfile.ubuntu

```diff
@@ -1,8 +1,7 @@
-
 FROM ubuntu:bionic
-MAINTAINER delivery-engineering@netflix.com
-COPY --from=compile /compiled_sources/echo-web/build/install/echo /opt/echo
+MAINTAINER sig-platform@spinnaker.io
+COPY echo-web/build/install/echo /opt/echo
 RUN apt-get update && apt-get -y install openjdk-11-jre-headless wget
 RUN adduser --disabled-login --system spinnaker
 USER spinnaker
-CMD ["/opt/echo/bin/echo"]
\ No newline at end of file
+CMD ["/opt/echo/bin/echo"]
```

Comment:
Nit: You may want to use ENTRYPOINT here and elsewhere, based on my understanding from https://goinbigdata.com/docker-run-vs-cmd-vs-entrypoint/

This would allow someone to run the container passing new/different JVM flags (I think we've seen users in the past trying to pass -D args to overrride how Java handles SSL handshakes and whatnot).

Not sure if people would or how they would pass them through Halyard, but it seems more correct. Feel free to ignore until we get someone actually wanting this.

Reply:
Hm, that's definitely interesting... I skimmed and I'm not sure I totally get it, but it seems like you're right...

but that's really unrelated to these PRs (this line only got changed because `vim` added a newline) so I don't think this is the right time to change it anyway.